ChordButton.elm

```diff
@@ -7,11 +7,11 @@ import Chords
 import Graphics.Input exposing(..)
 import Graphics.Element exposing (..)
 
-type alias Model = ((Note, Quality), ChordChart)
+type alias Model = (Note, Quality)
 init : Model
-init = 
-    ((B, Min), Chords.bm)
+init = (B, Min)
 
 view : Signal.Address App.Page -> Model -> Element
```

Comment:
I think this would be easier to read if `Model` => `Chord`, since we're exposing `view` upwards to the router